src/http_client_async.zig

```diff
@@ -1414,6 +1468,7 @@ disable_keepalive: bool = false,
 disable_decompression: bool = false,
 state: InternalState = .{},
 
+tls_props: ?SSLConfig = null,
```

Comment:
I suggest making this a pointer and allocating instead, as it increases the size of this struct and the struct is already insanely huge

Reply:
will do

Reply:
done in https://github.com/oven-sh/bun/pull/6621/commits/2e5dad83727d5d1bd683504f0314f89053155c9e